src/Framework/TestCase.php

```diff
@@ -843,17 +843,15 @@ final public function dataName(): int|string
      */
     final public function dataSetAsString(): string
```

Comment:
since TestDox formatter does not use the "className::methodName <dataset>" I did not change the format in this method itself but created a new `dataSetAsFilterString` used only in the default-printer